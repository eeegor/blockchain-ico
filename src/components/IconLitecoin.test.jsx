import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import { IconLitecoin } from '.';

describe('IconLitecoin', () => {
	test('renders IconLitecoin', () => {
		const tree = renderer.create(<IconLitecoin />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
