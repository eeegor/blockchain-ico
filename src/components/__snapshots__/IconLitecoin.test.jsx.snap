// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconLitecoin renders IconLitecoin 1`] = `
<svg
  height={26}
  version="1.1"
  viewBox="0 0 26 26"
  width={26}
  xmlns="http://www.w3.org/2000/svg"
  xmlnsXlink="http://www.w3.org/1999/xlink"
>
  <defs>
    <linearGradient
      id="linearGradient-1"
      x1="92.1554943%"
      x2="0%"
      y1="0%"
      y2="100%"
    >
      <stop
        offset="0%"
        stopColor="#2B60FF"
      />
      <stop
        offset="100%"
        stopColor="#619BFF"
      />
    </linearGradient>
  </defs>
  <g
    fill="none"
    fillRule="evenodd"
    id="icon_litecoin"
    stroke="none"
    strokeWidth="1"
  >
    <polyline
      fill="url(#linearGradient-1)"
      fillRule="nonzero"
      id="Shape"
      points="11.9544996 18.4282672 13.1164927 14.0528355 15.8675464 13.0477598 16.5518325 10.4763353 16.5284687 10.4125468 13.8204478 11.4018656 15.7715921 4.05500031 10.2380794 4.05500031 7.68643252 13.642829 5.55598504 14.4211133 4.85202999 17.0720833 6.98084744 16.294451 5.47698299 21.9449997 20.2038574 21.9449997 21.14797 18.4282672 11.9544996 18.4282672"
    />
  </g>
</svg>
`;
